src/engine/TransitivePathImpl.h

```diff
@@ -19,15 +19,17 @@ namespace detail {
 // the correct lifetime).
 template <typename ColumnType>
 struct TableColumnWithVocab {
-  const IdTable* table_;
+  std::optional<IdTableView<0>> table_;
```

Comment:
The names here seem to be off.
The first member is `PayloadColumns payload_`;
The second is something like `ColumnType startNodes_`